packages/big-design/src/components/TableNext/RowContainer/RowContainer.tsx

```diff
@@ -89,6 +93,15 @@ const InternalRowContainer = <T extends TableItem>({
             />
           );
         })}
+      {isExpanded && childrenRows !== undefined && hasHelperRowComponent && (
+        <tr key={`extra-helper-row-${parentRowIndex}`}>
+          <DataCell checkEmptyCell={true} colSpan={10000} width={100}>
```

Comment:
Few notes here:
1. Can we calculate the `colSpan` instead of using a magic number?
2. What does `width={100}` represent (another magic number)?
```suggestion
          <DataCell checkEmptyCell={true} colSpan={columns.length + <isSelectable> + <isDragable>} width={REPRESENTATION_OF_WIDTH}>
```

Reply:
1. oops forgot to remove the width 😅, good catch! Was trying to figure it out how to make the cell bigger.  Thanks!
2. I set a big number since didn't want to complicate too much the logic of calculating if columns + extra cell. I'll try to calculated it, and see how is gonna look 👍🏼  Thanks!